Doc/library/concurrent.futures.rst

```diff
@@ -54,11 +54,16 @@ Executor Objects
        specified by setting *chunksize* to a positive integer. For very long
        iterables, using a large value for *chunksize* can significantly improve
        performance compared to the default size of 1. With :class:`ThreadPoolExecutor`,
-       *chunksize* has no effect.
+       *chunksize* has no effect. By default, a reasonable number of tasks are
+       queued beyond the number of workers, an explicit *prefetch* count may be
+       provided to specify how many extra tasks should be queued.
 
```

Comment:
Actually, since this patch delays the calls to submit, if some other task is submitted midway through processing, it will be queued to go right after the most recently queued map work, but before any additional map work is queued (as results are processed and new tasks submitted). Queueing fewer tasks than the number of workers doesn't make a whole lot of sense to me, but this patch would allow intermingling map tasks with other tasks.